webhooks/tasks.py

```diff
@@ -0,0 +1,71 @@
+"""
+Celery tasks.
+"""
+import hashlib
+import hmac
+import json
+import logging
+
+from django.apps import apps
+from django.utils.encoding import force_bytes
+import requests
+
+from ccxcon.celery import async
+from webhooks.models import Webhook
+
+log = logging.getLogger(__name__)
+
+
+@async.task
+def publish_webhook(model_str, pk):
+    """
+    Publishes model updates to various services as defined by Webhook instances.
```

Comment:
Can you add the parameters and return value to the docstring in the napoleon format?